x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -54,273 +56,335 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
           injectReferences({ logger, params, savedObjectReferences }),
       },
       async executor(options) {
-        const {
-          alertId,
-          params,
-          previousStartedAt,
-          startedAt,
-          services,
-          spaceId,
-          state,
-          updatedBy: updatedByUser,
-          rule,
-        } = options;
-        let runState = state;
-        const { from, maxSignals, meta, ruleId, timestampOverride, to } = params;
-        const { alertWithPersistence, savedObjectsClient, scopedClusterClient } = services;
-        const searchAfterSize = Math.min(maxSignals, DEFAULT_SEARCH_AFTER_PAGE_SIZE);
-
-        const esClient = scopedClusterClient.asCurrentUser;
-
-        const ruleStatusClient = ruleExecutionLogClientOverride
-          ? ruleExecutionLogClientOverride
-          : new RuleExecutionLogClient({
-              underlyingClient: config.ruleExecutionLog.underlyingClient,
-              savedObjectsClient,
-              eventLogService,
-              logger,
-            });
+        agent.setTransactionName(`${options.rule.ruleTypeId} rule execution`);
```

Comment:
nit: Probably don't need `rule` here since all the [`ruleTypeId`'s](https://github.com/elastic/kibana/blob/117efdf87f46058c244c868d6b298ffdee8a9468/packages/kbn-securitysolution-rules/src/rule_type_constants.ts#L18-L23) end with `Rule` anyway?

```suggestion
        agent.setTransactionName(`${options.rule.ruleTypeId} execution`);
```

Reply:
Agree, changed!